engines/python/setup/djl_python/deepspeed.py

```diff
@@ -140,11 +139,10 @@ def _parse_properties(self, properties):
         self.model_dir = properties.get("model_dir")
         self.model_id = properties.get("model_id")
         self.task = properties.get("task")
-        self.data_type = get_torch_dtype_from_str(properties.get("data_type", "fp32"))
+        self.data_type = get_torch_dtype_from_str(properties.get("dtype", "fp16"))
```

Comment:
Why use fp16 as default?

Reply:
fp16 works better and better range of supported model on DeepSpeed. model like Bloom is not working on fp32

Reply:
I mean use a normal user, they will assume fp32 is default. They only override it to fp16 intentionally

Reply:
most LLM saved on huggingface is under fp16